lib/state/theme_state_notifier.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class ThemeStateNotifier extends StateNotifier<bool> {
```

Comment:
I don't see this as a useful class right now. Or are you going to hook it up with firestore later and manage storing theme into it? Then it would make sense, I guess.

Reply:
True for database fetching and saving